_articles/ar/legal.md

```diff
@@ -2,157 +2,207 @@
 lang: ar
 direction: rtl
 title: الجانب القانوني من المصادر المفتوحة
-description: كل ما تساءلت عنه بخصوص الجانب القانوني للمصادر المفتوحة، وأشياء إضافية لم تتساءل عنها.
+description: >-
+  كل ما تساءلت عنه بخصوص الجانب القانوني للمصادر المفتوحة، وأشياء إضافية لم
+  تتساءل عنها.
 class: legal
+toc:
+  لماذا-يهتم-الناس-بالجانب-القانوني-للمصادر-المفتوحة-بشكل-كبير-: لماذا يهتم الناس كثيرًا بالجانب القانوني للمصادر المفتوحة؟
+  هل-المشاريع-العامة-على-github-مفتوحة-المصدر: هل المشاريع العامة على GitHub مفتوحة المصدر؟
+  ما-هي-الأمور-التي-أحتاجها-لحماية-مشروعي-باختصار: ما هي الأمور التي أحتاجها لحماية مشروعي باختصار
+  ما-هي-رخصة-المصادر-المفتوحة-لمناسبة-لمشروعي: ما هي رخصة المصادر المفتوحة لمناسبة لمشروعي؟
+  ماذا-لو-أردت-أن-أغير-رخصة-مشروعي: ماذا لو أردت أن أغير رخصة مشروعي؟
+  هل-يحتاج-مشروعي-لاتفاقات-اضافية-مع-المساهمين: هل يحتاج مشروعي لاتفاقات اضافية مع المساهمين؟
+  ما-الذي-يحتاج-الفريق-القانوني-لشركتي-معرفته: ما الذي يحتاج الفريق القانوني لشركتي معرفته؟
```

Comment:
This section (`toc`) is not needed now.

Reply:
why?

Reply:
Not sure :grin:
Check https://github.com/github/opensource.guide/pull/779 for more details.